README.md

```diff
@@ -0,0 +1,38 @@
+# Money to Prisoners Cashbook
+The Cashbook UI for the Money to Prisoners Project
+
+## Dependencies
+### Docker
+To run this project locally you need to have 
+[docker](http://docs.docker.com/installation/mac/) and 
+[docker-compose](https://docs.docker.com/compose/install/) installed.
+
+### Other Repositories
+Alongside this repository you'll need the [API server](https://github.com/ministryofjustice/money-to-prisoners-api)
+and if you're planning to deploy then you'll need the [deployment repository](https://github.com/ministryofjustice/money-to-prisoners-deploy)
+
+### Development Server
```

Comment:
Or possibly tweak this to be

- Working with this repo
  - Dependencies: docker, docker compose, boot2docker
  - Development server: [the instructions below]